entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
The EF6 warning shouldn't be the first content, it should come after the intro. That's what I did in my rewrite. It should be an [INCLUDE] so it's only written once, not copy/paste everywhere.

>I don't like the using-in-comment approach

I don't really like it either, it should only be used in non-beginner articles. We're seeing many non-Visual Studio customers struggle with finding the right using. The code comment is the least obtrusive but it is ugly. Where I've employed it, customer success rate has increased.

Given this is low level info it's probably worth calling out explicitly - but not and afterthought obtrusive [!NOTE]